Systems/c172p-damage.xml

```diff
@@ -243,38 +243,39 @@
             <test logic="AND" value="1.0">
                 /velocities/airspeed-kt GT damage/limits-vne-high
             </test>
+            
             <test logic="AND" value="1.0">
                 /velocities/airspeed-kt GT damage/limits-vne-medium
                 damage/roll-moment GT 4000.0
-                wing-damage/right-wing GT 0.0
+                wing-damage/right-wing LT 1.0
```

Comment:
Yes, that's on purpose. Rather, that _was_. Initially.
`This was in order to avoid breaking the right wing as soon as the left one was broken (e.g. by a roll moment)`

`But, from recent tests, I realize that it seems useless, the right wing holds as long as the `limits-vne-high` is not reached. I think that this condition can be deleted too.`
Thus I've removed it.